analyze.js

```diff
@@ -104,19 +104,72 @@ async function attachScanFiles(scanGlobs, formData) {
   }
 }
 
+async function validateBranches(branches, config) {
+  const defaultBranch = branches.filter(branch => branch.isDefault)[0].name
+
+  // If target branch is not defined, use the default branch. This also ensures even if
+  // base branch is defined, the default branch is not created as a child of the base branch
+  if (typeof config.targetBranchName == 'undefined') {
+    core.info(`No target branch was defined. Using default target branch '${defaultBranch}'`)
+    config.targetBranchName = defaultBranch
+    config.baseBranchName = undefined
+  }
+
+  const baseBranchPresent = srmClient.checkIfBranchPresent(branches, config.baseBranchName)
+  const targetBranchPresent = srmClient.checkIfBranchPresent(branches, config.targetBranchName)
```

Comment:
The use of `srmClient` makes it seem like some HTTP request will be done here, when it's really just doing some filtering on `branches`

I suggest renaming `srmClient` to `srm`, or import that module's members individually via destructuring with `const { SrmApiClient, checkIfBranchPresent } = require('./srm')`